packages/eslint-config-react-app/package.json

```diff
@@ -15,17 +15,20 @@
     "index.js"
   ],
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "2.x",
-    "@typescript-eslint/parser": "2.x",
+    "@typescript-eslint/eslint-plugin": "3.x",
+    "@typescript-eslint/parser": "3.x",
     "babel-eslint": "10.x",
-    "eslint": "6.x",
-    "eslint-plugin-flowtype": "3.x || 4.x",
+    "eslint": "7.x",
+    "eslint-plugin-flowtype": "5.x",
     "eslint-plugin-import": "2.x",
     "eslint-plugin-jsx-a11y": "6.x",
     "eslint-plugin-react": "7.x",
-    "eslint-plugin-react-hooks": "1.x || 2.x || 3.x || 4.x"
+    "eslint-plugin-react-hooks": "4.x"
   },
   "dependencies": {
     "confusing-browser-globals": "^1.0.9"
+  },
+  "engines": {
```

Comment:
I'm not sure we need to specify `engines` here. The dependencies should have declared that restriction if it exists already, no? Thoughts?

Reply:
I think it's always a good idea to specify them yourself, since that makes it clear right away + `yarn install` will fail if you don't have the right `node` version